pyracing/client.py

```diff
@@ -73,13 +68,16 @@ async def authenticate(self):
                 'visiting members.iracing.com'
             )
         else:
+            self.initial_auth = True
             self.log.info('Login successful')
 
-    # Wrapper for all functions that builds the final self.session.get()
-
-    async def __build_request(self, url, params):
+    async def _build_request(self, url, params):
```

Comment:
Learned that two underscores is for name mangling, one underscore is for private methods